core/shared/src/main/scala/zio/logging/package.scala

```diff
@@ -299,4 +313,25 @@ package object logging {
     })
     stringLogger
   }
+
+  private val metricLogger = new ZLogger[String, Unit] {
+    override def apply(
+      trace: Trace,
+      fiberId: FiberId,
+      logLevel: LogLevel,
+      message: () => String,
+      cause: Cause[Any],
+      context: FiberRefs,
+      spans: List[LogSpan],
+      annotations: Map[String, String]
+    ): Unit = {
+      loggedTotalMetrics.get(logLevel).foreach { counter =>
+        val tags = context.get(FiberRef.currentTags).getOrElse(Set.empty)
+        counter.unsafe.update(1, tags)(Unsafe.unsafe)
+      }
+      ()
+    }
+  }
+
+  val metrics: ZLayer[Any, Nothing, Unit] = Runtime.addLogger(metricLogger)
```

Comment:
Maybe `logMetrics` to avoid clashing with identical names in scope?

Reply:
ok, that is good idea